src/main/java/core/basesyntax/Storage.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> fruitStorage = new HashMap<>();
+
+    public static Map<String, Integer> getFruitStorage() {
+        return fruitStorage;
+    }
+
+    public static void addFruit(String fruit, int quantity) {
+        fruitStorage.put(fruit, fruitStorage.getOrDefault(fruit, 0) + quantity);
+    }
+
+    public static void removeFruit(String fruit, int quantity) {
```

Comment:
Consider checking for negative quantities in `removeFruit` method. If the resulting quantity of fruit becomes negative, you should throw a `RuntimeException` according to the checklist item for handling Purchase operation.